features/facebookImport/test/utils/importer-assertions.js

```diff
@@ -8,18 +8,19 @@ import {
     IMPORT_WARNING,
 } from "../../src/model/importers/utils/importer-status";
 
-export function expectError(result, errorClass) {
+export function expectError(result, errorClass, importerClass) {
     expect(result.status).toBe(IMPORT_ERROR);
     expect(result.message).toBe(errorClass.name);
     expect(result.error.name).toBe(errorClass.name);
+    expect(result.importerClass).toBe(importerClass);
```

Comment:
OK, I see the point of this here, but as a matter of fact, name is a one-to-one mapping to importerClass, so unless that class has some more attributes you'll be testing testing the name might be more than enough.

Reply:
The initial idea was that we might take more information from the importer class, but at the moment we just have the name. As a first step this PR makes things consistent, as now building the report can fail if there are import warnings.

Reply:
Right, but this is the class, not an instance. Unless you want to instantiate an object from it, I don't see what functionality that would have over the name.

Reply:
Initially I think we wanted to put some static data on the class side, like what the importer should import, but that is not the case now. We use also the class object in the status of running analyses. 
For the moment these changes avoid a crash in `DataImportingStatusAnalysis`, and afterwards yes, we could replace it with the class name in both the status of analyses and importers.